src/components/utils/Sort.ts

```diff
@@ -39,10 +39,10 @@ function sort(a: string, b: string) {
     const bArr = b.split('.').map(Number);
     const len = Math.max(aArr.length, bArr.length);
     for (let i = 0; i < len; i++) {
-        if (aArr[i] === undefined) {
+        if (i > aArr.length - 1) {
```

Comment:
```suggestion
        if (i >= aArr.length) {
```
How about change it to `>=`?

Reply:
@iskey yes this is better. will change this.